3737/2/model/user_class.rb

```diff
@@ -0,0 +1,16 @@
+require 'ohm'
+
+# user model
+class User < Ohm::Model
+  attribute :number
+  attribute :in_camp
+
+  def self.registered(id)
+    self[id]
+  end
+
+  def self.find_user(number)
+    student_number = number
+    User.all.select { |_, hash| hash[number] == student_number }
```

Comment:
Do you have multiple users with same student number? I don't think so. I think `.find` is better in such case